src/routes/(admin)/questionbank/+page.svelte

```
@@ -1,35 +1,140 @@
+						>
+							Clear all filters
+						</button>
+					</div>
+				</div>
+			`;
+		}
+		return null;
+	}
+
+	// Render expanded row content
+	function expandedRowContent(question: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)